ada_feeding_msgs/msg/AcquisitionSchema.msg

```diff
@@ -0,0 +1,59 @@
+# This message is used to specify an Acquisition Action based
+# on the schema available at:
+# https://drive.google.com/file/d/1IP74g9bTQAbf8lahd7nxqTfV1Aook3bG/view
+
+### Reference Frames
+
+## Food Frame:
+# Origin: Center, Bottom of the food
+# +Z = against gravity
+# +X = major axis of food bounding ellipse
+
+## Approach Frame
+# Same as Food Frame, except:
+# +X = -(pre_transform.translation())
+# Projected onto Food X/Y plane
+
+## Utensil Frame = "forkTip" TF
+
```

Comment:
See comment in `acquisition_library.yaml` re. a unique name/id per action.

Reply:
See reply below